templates/nunjucks/base.njk.html

```diff
@@ -53,6 +53,9 @@
           Ministry of Justice
         </a>
 
+        <a class="moj-header__link moj-header__link--service-name"
+        href="{{ flowUrl }}">MOJ Online Editor</a>
```

Comment:
Are we using `MoJ Online Editor` now or are we keeping `Form Builder Console`?

Reply:
We are using MoJ Online Editor

Reply:
okey dokey. if that's the case we should make that consistent across our platform and tools